python/tvm/relay/frontend/tflite.py

```diff
@@ -692,6 +773,11 @@ def _hard_swish(data):
 
     def convert_relu6(self, op):
         """Convert TFLite ReLU6"""
+        try:
+            from tflite.ActivationFunctionType import ActivationFunctionType
```

Comment:
Same as relu,  I think this is unnecessary given the import of ActivationFunctionType in the constructor [here](https://github.com/apache/incubator-tvm/blob/b979bf6a7630ada055fff2d65e1cd0f8d55bb6a0/python/tvm/relay/frontend/tflite.py#L54)

Reply:
Same as before